src/core/index.js

```diff
@@ -62,6 +63,7 @@ class IPFS {
     this.bitswap = components.bitswap(this)
     this.ping = components.ping(this)
     this.pubsub = components.pubsub(this)
+    this.repoExists = utils.repoExists(this)
```

Comment:
I'll give that a shot. The test should still run agains this.repoExists though, right?